components/molecules/PageBreadcrumbs.jsx

```diff
@@ -0,0 +1,129 @@
+            path: config.path
+        };
+
+        // Set initial data if necessary
+        const initialData = config.path === '/' ? [initialBreadcrumb] : [breadcrumbConfig];
+
+        const searchBreakpoints = (pathSections, subpaths, accumulator = []) =>
+            pathSections.reduce((trail, pathSection, index) => {
+                // Find current path subpathData
+                const subpathData = subpaths && subpaths.find(({ path }) => path === pathSection);
+                // Find section param subpathData if no exact path was found
+                const paramData =
+                    !subpathData && subpaths && subpaths.find(({ path }) => path.charAt(0) === ':');
+                const breadcrumbData = subpathData || paramData || { path: pathSection };
+                const acceptBreadcrumb = trail.length < maxLength;
+                // Push section subpathData  to accumulator
+                if (acceptBreadcrumb && breadcrumbData) {
+                    // Set defaults depending on type of path (param or regular path)
+                    const breadcrumbPath = paramData ? pathSection : breadcrumbData.path;
+                    const breadcrumbTitle =
+                        breadcrumbData.title || this._getParamTitle(pathSection);
+                    // Build full path for link
+                    const currentPath = this._buildHref(trail, breadcrumbPath);
+                    trail.push({
+                        ...breadcrumbData,
+                        path: currentPath,
+                        title: breadcrumbTitle
+                    });
+                }
+                const trailContinues = trail.length < maxLength;
+                // Go deeper if there are subpaths
+                if (breadcrumbData && breadcrumbData.subpaths && trailContinues) {
+                    const nextIndex = index + 1;
+                    const remainingParts = pathSections.slice(nextIndex);
+                    // Search for the remaining pathname parts on the current path's subpaths,
+                    return searchBreakpoints(remainingParts, breadcrumbData.subpaths, trail);
+                }
+                return trail;
+            }, accumulator);
+
+        // Divide pathname into sections
+        const parts = pathname.split('/').filter(Boolean);
+        const duplicateRoot = parts[0] === config.path;
+        const initialSections = duplicateRoot ? parts.slice(1) : parts;
+        const maxLength = initialSections.length + 1;
+        // Compare pathname parts against user provided configuration
+        const breadcrumbs = searchBreakpoints(initialSections, config.subpaths, initialData);
+        return breadcrumbs;
+    };
+
+    render() {
+        const { config, linkComponent, hideRoot, pathname } = this.props;
+        const breadcrumbs = this._findTrail(pathname, config);
+        const displayBreadcrumbs =
+            hideRoot && breadcrumbs.length > 1 ? breadcrumbs.slice(1) : breadcrumbs;
```

Comment:
Since we set a config at the default props, there is technically always a config being passed. As a solution for the additional slash, we check to see if the title is "/" and if it is we will skip it.